From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Mon, 16 May 2022 17:59:02 +0800
Subject: build: Drop gtk3-install.[vsprops|props]

They are not used anymore, and are removed at this point.

Origin: upstream, 3.24.35, commit:f4811ea1f1106b34b9980c2566ccd10374dea798
---
 gdk/Makefile.am                  |   8 --
 gdk/broadway/Makefile.am         |   8 +-
 gdk/win32/Makefile.am            |   8 +-
 gtk/Makefile.am                  |   9 --
 libgail-util/Makefile.am         |   8 +-
 win32/Makefile.msvcproj          |  30 +-----
 win32/vs10/Makefile.am           |  16 ---
 win32/vs10/gtk3-install.propsin  | 222 ---------------------------------------
 win32/vs9/Makefile.am            |  11 --
 win32/vs9/gtk3-install.vspropsin | 208 ------------------------------------
 10 files changed, 4 insertions(+), 524 deletions(-)
 delete mode 100644 win32/vs10/gtk3-install.propsin
 delete mode 100644 win32/vs9/gtk3-install.vspropsin

diff --git a/gdk/Makefile.am b/gdk/Makefile.am
index 632c687..5eae5a1 100644
--- a/gdk/Makefile.am
+++ b/gdk/Makefile.am
@@ -458,13 +458,6 @@ MSVCPROJS = gdk-3
 
 gdk_3_FILES = $(gdk_c_sources)
 gdk_3_EXCLUDES = dummy
-gdk_3_HEADERS_DIR = $(gdkincludedir)
-
-gdk_3_HEADERS_INST = \
-	$(gdkinclude_HEADERS)		\
-	$(deprecatedinclude_HEADERS)
-
-gdk_3_HEADERS_EXCLUDES = dummy
 
 include $(top_srcdir)/win32/Makefile.msvcproj
 
@@ -509,7 +502,6 @@ endif
 
 dist-hook: \
 	$(top_builddir)/win32/vs9/gdk-3.vcproj	\
-	$(top_builddir)/win32/vs9/gdk-3.headers	\
 	$(INTROSPECTION_INTERMEDIATE_ITEMS)
 
 DISTCLEANFILES = gdkconfig.h stamp-gc-h
diff --git a/gdk/broadway/Makefile.am b/gdk/broadway/Makefile.am
index 81725de..3b51870 100644
--- a/gdk/broadway/Makefile.am
+++ b/gdk/broadway/Makefile.am
@@ -72,11 +72,6 @@ MSVCPROJS = gdk3-broadway broadwayd
 
 gdk3_broadway_FILES = $(libgdk_broadway_la_SOURCES)
 gdk3_broadway_EXCLUDES = dummy
-gdk3_broadway_HEADERS_DIR = $(libgdkbroadwayincludedir)
-
-gdk3_broadway_HEADERS_INST = $(libgdkbroadwayinclude_HEADERS)
-
-gdk3_broadway_HEADERS_EXCLUDES = dummy
 
 broadwayd_FILES = $(broadwayd_SOURCES)
 broadwayd_EXCLUDES = dummy
@@ -85,7 +80,6 @@ include $(top_srcdir)/win32/Makefile.msvcproj
 
 dist-hook: \
 	$(top_builddir)/win32/vs9/gdk3-broadway.vcproj		\
-	$(top_builddir)/win32/vs9/broadwayd.vcproj		\
-	$(top_builddir)/win32/vs9/gdk3-broadway.headers
+	$(top_builddir)/win32/vs9/broadwayd.vcproj
 
 -include $(top_srcdir)/git.mk
diff --git a/gdk/win32/Makefile.am b/gdk/win32/Makefile.am
index a2d52ce..d5fb09a 100644
--- a/gdk/win32/Makefile.am
+++ b/gdk/win32/Makefile.am
@@ -114,16 +114,10 @@ MSVCPROJS = gdk3-win32
 
 gdk3_win32_FILES = $(libgdk_win32_la_SOURCES)
 gdk3_win32_EXCLUDES = dummy
-gdk3_win32_HEADERS_DIR = $(libgdkwin32includedir)
-
-gdk3_win32_HEADERS_INST = $(libgdkwin32include_HEADERS)
-
-gdk3_win32_HEADERS_EXCLUDES = dummy
 
 include $(top_srcdir)/win32/Makefile.msvcproj
 
 dist-hook: \
-	$(top_builddir)/win32/vs9/gdk3-win32.vcproj	\
-	$(top_builddir)/win32/vs9/gdk3-win32.headers
+	$(top_builddir)/win32/vs9/gdk3-win32.vcproj
 
 -include $(top_srcdir)/git.mk
diff --git a/gtk/Makefile.am b/gtk/Makefile.am
index 1b66241..e96afd3 100644
--- a/gtk/Makefile.am
+++ b/gtk/Makefile.am
@@ -528,14 +528,6 @@ gtk_3_FILES = \
 	$(gtk_use_win32_c_sources)
 
 gtk_3_EXCLUDES = dummy
-gtk_3_HEADERS_DIR = $(gtkincludedir)
-
-gtk_3_HEADERS_INST = \
-	$(filter-out gtkversion.h, $(gtkinclude_HEADERS))	\
-	$(a11y_h_sources)	\
-	$(gtk_deprecated_h_sources)
-
-gtk_3_HEADERS_EXCLUDES = gtktypebuiltins.h
 
 include $(top_srcdir)/win32/Makefile.msvcproj
 
@@ -592,7 +584,6 @@ endif
 
 dist-hook: \
 	$(top_builddir)/win32/vs9/gtk-3.vcproj	\
-	$(top_builddir)/win32/vs9/gtk-3.headers	\
 	$(INTROSPECTION_INTERMEDIATE_ITEMS)
 
 # Install a RC file for the default GTK+ theme, and key themes
diff --git a/libgail-util/Makefile.am b/libgail-util/Makefile.am
index aa2f93a..0b389ad 100644
--- a/libgail-util/Makefile.am
+++ b/libgail-util/Makefile.am
@@ -82,16 +82,10 @@ MSVCPROJS = gailutil-3
 gailutil_3_FILES = $(libgailutil_3_la_SOURCES)
 
 gailutil_3_EXCLUDES = dummy
-gailutil_3_HEADERS_DIR = $(libgailutilincludedir)
-
-gailutil_3_HEADERS_INST = $(libgailutilinclude_HEADERS)
-
-gailutil_3_HEADERS_EXCLUDES = dummy
 
 include $(top_srcdir)/win32/Makefile.msvcproj
 
 dist-hook: \
-	$(top_builddir)/win32/vs9/gailutil-3.vcproj	\
-	$(top_builddir)/win32/vs9/gailutil-3.headers
+	$(top_builddir)/win32/vs9/gailutil-3.vcproj
 
 -include $(top_srcdir)/git.mk
diff --git a/win32/Makefile.msvcproj b/win32/Makefile.msvcproj
index cd7e469..c341313 100644
--- a/win32/Makefile.msvcproj
+++ b/win32/Makefile.msvcproj
@@ -17,13 +17,9 @@
 #   MSVCPROJS = YourProject (can be multiple projects in a single srcdir)
 #   YourProject_FILES = $(libyourlib_1_0_SOURCES)
 #   YourProject_EXCLUDES = ... # list of sources to exclude, separated by '|', wildcards allowed; use random unsed value if none
-#   YourProject_HEADERS_DIR = $(libyourlibincludedir)
-#   YourProject_HEADERS_INST = $(libyourlib_1_0_HEADERS)
-#   YourProject_HEADERS_EXCLUDES = ... # <list of headers to exclude from installation, separated by '|', wildcards allowed; use random unsed value if none>
 #
 #   dist-hook: \ # (or add to it if it is already there, note the vs9 items will also call the vs10 items in the process)
-#   	$(top_builddir)/win32/vs9/YourProject.vcproj	\
-#   	$(top_builddir)/win32/vs9/YourProject.headers
+#   	$(top_builddir)/win32/vs9/YourProject.vcproj
 
 
 # Private functions
@@ -40,13 +36,6 @@ _proj_files_raw=$(subst /,\\,$($(_proj_name)_FILES))
 _proj_files=$(subst $(srcdir)\\,,$(subst $(builddir)\\,,$(subst $(top_builddir)\\$(_proj_path)\\,\\,$(_proj_files_raw))))
 _proj_filters=$($(_proj_name)_EXCLUDES)
 
-_proj_headers_raw=$(subst /,\\,$($(_proj_name)_HEADERS_INST))
-_proj_headers=$(subst $(srcdir)\\,,$(subst $(builddir)\\,,$(subst $(top_builddir)\\$(_proj_path)\\,\\,$(_proj_headers_raw))))
-_proj_headers_excludes=$($(_proj_name)_HEADERS_EXCLUDES)
-
-_headers_dest_posix=$(subst $(includedir),,$($(_proj_name)_HEADERS_DIR))
-_headers_destdir=$(subst /,\\,$(_headers_dest_posix))
-
 #
 # Creates Visual Studio 2008/2010 projects from items passed in from autotools files
 # $(1) - Base Name of the MSVC project files (outputs)
@@ -95,23 +84,6 @@ $(top_builddir)/win32/vs9/$(1).vcproj: Makefile
 	$(RM) $(1).vs10.sourcefiles
 	$(RM) $(1).vs10.sourcefiles.filters
 
-$(top_builddir)/win32/vs10/$(1).vs10.headers: $(top_builddir)/win32/vs9/$(1).headers
-
-$(top_builddir)/win32/vs9/$(1).headers: Makefile
-	-$(RM) $(top_builddir)/win32/vs9/$(1).headers
-	-$(RM) $(top_builddir)/win32/vs10/$(1).vs10.headers
-
-	for F in $(_proj_headers); do \
-		case $$$$F in \
-		$(_proj_headers_excludes)) \
-			;; \
-		*.h|*.hpp|*.hh|*.hxx) \
-			echo 'copy ..\..'$(_proj_subdir)$$$$F' $$$$(CopyDir)\include'$(_headers_destdir)'\'$$$$F'&#x0D;&#x0A;' >>$(top_builddir)/win32/vs9/$(1).headers && \
-			echo 'copy ..\..'$(_proj_subdir)$$$$F' $$$$(CopyDir)\include'$(_headers_destdir)'\'$$$$F >>$(top_builddir)/win32/vs10/$(1).vs10.headers \
-			;; \
-		esac;	\
-	done
-
 endef
 
 $(foreach proj,$(MSVCPROJS),$(eval $(call msvcproj-builder,$(proj))))
diff --git a/win32/vs10/Makefile.am b/win32/vs10/Makefile.am
index ef62710..c226d5a 100644
--- a/win32/vs10/Makefile.am
+++ b/win32/vs10/Makefile.am
@@ -22,13 +22,6 @@ GENERATED_ITEMS = \
 	gtk3-install.props			\
 	gtk3-version-paths.props
 
-MSVC10_HEADERS_LISTS =	\
-	gdk3-win32.vs10.headers		\
-	gdk3-broadway.vs10.headers	\
-	gdk-3.vs10.headers		\
-	gtk-3.vs10.headers		\
-	gailutil-3.vs10.headers
-
 EXTRA_DIST += \
 	README.txt				\
 	gtk+.sln				\
@@ -72,13 +65,4 @@ EXTRA_DIST += \
 DISTCLEANFILES = \
 	$(GENERATED_ITEMS)
 
-gtk3-install.props: $(top_srcdir)/win32/vs10/gtk3-install.propsin $(MSVC10_HEADERS_LISTS)
-	-$(RM) $(top_builddir)/win32/vs11/gtk3-install.props
-	-$(RM) $(top_builddir)/win32/vs12/gtk3-install.props
-	-$(RM) $(top_builddir)/win32/vs14/gtk3-install.props
-	-$(RM) $(top_builddir)/win32/vs15/gtk3-install.props
-	-$(RM) $(top_builddir)/win32/vs16/gtk3-install.props
-	$(CPP) -P - <$(top_srcdir)/win32/vs10/gtk3-install.propsin >$@
-	rm $(MSVC10_HEADERS_LISTS)
-
 -include $(top_srcdir)/git.mk
diff --git a/win32/vs10/gtk3-install.propsin b/win32/vs10/gtk3-install.propsin
deleted file mode 100644
index 7e6cc46..0000000
--- a/win32/vs10/gtk3-install.propsin
+++ /dev/null
@@ -1,222 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ImportGroup Label="PropertySheets">
-    <Import Project="gtk3-build-defines.props" />
-  </ImportGroup>
-  <PropertyGroup Label="UserMacros">
-    <BinDir>$(SolutionDir)$(Configuration)\$(Platform)\bin</BinDir>
-    <GtkDoInstallBin>
-mkdir $(CopyDir)\bin
-mkdir $(CopyDir)\share\gir-1.0
-mkdir $(CopyDir)\lib\pkgconfig
-mkdir $(CopyDir)\lib\girepository-1.0
-
-copy "$(BinDir)\$(GtkDllPrefix)gdk-3$(GtkDllSuffix).dll" $(CopyDir)\bin
-copy "$(BinDir)\$(GtkDllPrefix)gdk-3$(GtkDllSuffix).pdb" $(CopyDir)\bin
-copy "$(BinDir)\gdk-$(ApiVersion).lib" $(CopyDir)\lib
-
-if "$(Configuration)" == "Release_Broadway" goto DO_BROADWAY_BIN
-if "$(Configuration)" == "Debug_Broadway" goto DO_BROADWAY_BIN
-copy "$(BinDir)\$(GtkDllPrefix)gtk-3$(GtkDllSuffix).dll" $(CopyDir)\bin
-copy "$(BinDir)\$(GtkDllPrefix)gtk-3$(GtkDllSuffix).pdb" $(CopyDir)\bin
-copy "$(BinDir)\$(GtkDllPrefix)gailutil-3$(GtkDllSuffix).dll" $(CopyDir)\bin
-copy "$(BinDir)\$(GtkDllPrefix)gailutil-3$(GtkDllSuffix).pdb" $(CopyDir)\bin
-copy "$(BinDir)\gtk-$(ApiVersion).lib" $(CopyDir)\lib
-copy "$(BinDir)\gailutil-$(ApiVersion).lib" $(CopyDir)\lib
-copy "$(BinDir)\gtk3-demo.exe" $(CopyDir)\bin
-copy "$(BinDir)\gtk3-demo.pdb" $(CopyDir)\bin
-copy "$(BinDir)\gtk3-demo-application.exe" $(CopyDir)\bin
-copy "$(BinDir)\gtk3-demo-application.pdb" $(CopyDir)\bin
-copy "$(BinDir)\gtk3-icon-browser.exe" $(CopyDir)\bin
-copy "$(BinDir)\gtk3-icon-browser.pdb" $(CopyDir)\bin
-copy "$(BinDir)\gtk-encode-symbolic-svg.exe" $(CopyDir)\bin
-copy "$(BinDir)\gtk-encode-symbolic-svg.pdb" $(CopyDir)\bin
-copy "$(BinDir)\gtk-update-icon-cache.exe" $(CopyDir)\bin
-copy "$(BinDir)\gtk-update-icon-cache.pdb" $(CopyDir)\bin
-copy "$(BinDir)\gtk-query-settings.exe" $(CopyDir)\bin
-copy "$(BinDir)\gtk-query-settings.pdb" $(CopyDir)\bin
-copy "$(BinDir)\gtk-builder-tool.exe" $(CopyDir)\bin
-copy "$(BinDir)\gtk-builder-tool.pdb" $(CopyDir)\bin
-if exist $(BinDir)\Gdk-3.0.gir copy "$(BinDir)\Gdk-3.0.gir" $(CopyDir)\share\gir-1.0
-if exist $(BinDir)\GdkWin32-3.0.gir copy "$(BinDir)\GdkWin32-3.0.gir" $(CopyDir)\share\gir-1.0
-if exist $(BinDir)\Gtk-3.0.gir copy "$(BinDir)\Gtk-3.0.gir" $(CopyDir)\share\gir-1.0
-if exist $(BinDir)\Gdk-3.0.typelib copy "$(BinDir)\Gdk-3.0.typelib" $(CopyDir)\lib\girepository-1.0
-if exist $(BinDir)\GdkWin32-3.0.typelib copy "$(BinDir)\GdkWin32-3.0.typelib" $(CopyDir)\lib\girepository-1.0
-if exist $(BinDir)\Gtk-3.0.typelib copy "$(BinDir)\Gtk-3.0.typelib" $(CopyDir)\lib\girepository-1.0
-goto DONE_BIN
-
-:DO_BROADWAY_BIN
-copy "$(BinDir)\broadwayd.exe" $(CopyDir)\bin
-copy "$(BinDir)\broadwayd.pdb" $(CopyDir)\bin
-
-if "$(Configuration)" == "Release_Broadway" goto DO_BROADWAY_RELEASE
-if "$(Configuration)" == "Debug_Broadway" goto DO_BROADWAY_DEBUG
-:DO_BROADWAY_RELEASE
-copy .\Release\$(Platform)\bin\$(GtkDllPrefix)gtk-3$(GtkDllSuffix).dll $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\$(GtkDllPrefix)gtk-3$(GtkDllSuffix).pdb $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\$(GtkDllPrefix)gailutil-3$(GtkDllSuffix).dll $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\$(GtkDllPrefix)gailutil-3$(GtkDllSuffix).pdb $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk-$(ApiVersion).lib $(CopyDir)\lib
-copy .\Release\$(Platform)\bin\gailutil-$(ApiVersion).lib $(CopyDir)\lib
-copy .\Release\$(Platform)\bin\gtk3-demo.exe $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk3-demo.pdb $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk3-demo-application.exe $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk3-demo-application.pdb $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk3-icon-browser.exe $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk3-icon-browser.pdb $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk-encode-symbolic-svg.exe $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk-encode-symbolic-svg.pdb $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk-update-icon-cache.exe $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk-update-icon-cache.pdb $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk-query-settings.exe $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk-query-settings.pdb $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk-builder-tool.exe $(CopyDir)\bin
-copy .\Release\$(Platform)\bin\gtk-builder-tool.pdb $(CopyDir)\bin
-if exist .\Release\$(Platform)\bin\Gdk-3.0.gir copy .\Release\$(Platform)\bin\Gdk-3.0.gir $(CopyDir)\share\gir-1.0
-if exist .\Release\$(Platform)\bin\GdkWin32-3.0.gir copy .\Release\$(Platform)\bin\GdkWin32-3.0.gir $(CopyDir)\share\gir-1.0
-if exist .\Release\$(Platform)\bin\Gtk-3.0.gir copy .\Release\$(Platform)\bin\Gtk-3.0.gir $(CopyDir)\share\gir-1.0
-if exist .\Release\$(Platform)\bin\Gdk-3.0.typelib copy .\Release\$(Platform)\bin\Gdk-3.0.typelib $(CopyDir)\lib\girepository-1.0
-if exist .\Release\$(Platform)\bin\GdkWin32-3.0.typelib copy .\Release\$(Platform)\bin\GdkWin32-3.0.typelib $(CopyDir)\lib\girepository-1.0
-if exist .\Release\$(Platform)\bin\Gtk-3.0.typelib copy .\Release\$(Platform)\bin\Gtk-3.0.typelib $(CopyDir)\lib\girepository-1.0
-
-goto DONE_BIN
-
-:DO_BROADWAY_DEBUG
-copy .\Debug\$(Platform)\bin\$(GtkDllPrefix)gtk-3$(GtkDllSuffix).dll $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\$(GtkDllPrefix)gtk-3$(GtkDllSuffix).pdb $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\$(GtkDllPrefix)gailutil-3$(GtkDllSuffix).dll $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\$(GtkDllPrefix)gailutil-3$(GtkDllSuffix).pdb $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk-$(ApiVersion).lib $(CopyDir)\lib
-copy .\Debug\$(Platform)\bin\gailutil-$(ApiVersion).lib $(CopyDir)\lib
-copy .\Debug\$(Platform)\bin\gtk3-demo.exe $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk3-demo.pdb $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk3-demo-application.exe $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk3-demo-application.pdb $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk3-icon-browser.exe $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk3-icon-browser.pdb $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk-encode-symbolic-svg.exe $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk-encode-symbolic-svg.pdb $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk-update-icon-cache.exe $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk-update-icon-cache.pdb $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk-query-settings.exe $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk-query-settings.pdb $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk-builder-tool.exe $(CopyDir)\bin
-copy .\Debug\$(Platform)\bin\gtk-builder-tool.pdb $(CopyDir)\bin
-if exist .\Debug\$(Platform)\bin\Gdk-3.0.gir copy .\Debug\$(Platform)\bin\Gdk-3.0.gir $(CopyDir)\share\gir-1.0
-if exist .\Debug\$(Platform)\bin\GdkWin32-3.0.gir copy .\Debug\$(Platform)\bin\GdkWin32-3.0.gir $(CopyDir)\share\gir-1.0
-if exist .\Debug\$(Platform)\bin\Gtk-3.0.gir copy .\Debug\$(Platform)\bin\Gtk-3.0.gir $(CopyDir)\share\gir-1.0
-if exist .\Debug\$(Platform)\bin\Gdk-3.0.typelib copy .\Debug\$(Platform)\bin\Gdk-3.0.typelib $(CopyDir)\lib\girepository-1.0
-if exist .\Debug\$(Platform)\bin\GdkWin32-3.0.typelib copy .\Debug\$(Platform)\bin\GdkWin32-3.0.typelib $(CopyDir)\lib\girepository-1.0
-if exist .\Debug\$(Platform)\bin\Gtk-3.0.typelib copy .\Debug\$(Platform)\bin\Gtk-3.0.typelib $(CopyDir)\lib\girepository-1.0
-
-:DONE_BIN
-
-copy ..\gdk-3.0.pc $(CopyDir)\lib\pkgconfig
-copy ..\gdk-3.0.pc $(CopyDir)\lib\pkgconfig\gdk-win32-3.0.pc
-copy "..\gtk+-3.0.pc" $(CopyDir)\lib\pkgconfig
-copy "..\gtk+-3.0.pc" "$(CopyDir)\lib\pkgconfig\gtk+-win32-3.0.pc"
-copy ..\gail-3.0.pc $(CopyDir)\lib\pkgconfig
-    </GtkDoInstallBin>
-    <GtkDoInstall>
-echo off
-mkdir $(CopyDir)\include\gtk-$(ApiVersion)\gdk\win32
-mkdir $(CopyDir)\include\gtk-$(ApiVersion)\gdk\deprecated
-mkdir $(CopyDir)\include\gtk-$(ApiVersion)\gtk\a11y
-mkdir $(CopyDir)\include\gtk-$(ApiVersion)\gtk\deprecated
-copy ..\..\gdk\win32\gdkwin32.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk
-
-if "$(Configuration)" == "Release" goto COPY_RELEASE_BUILT_HEADERS
-if "$(Configuration)" == "Release_Broadway" goto COPY_RELEASE_BUILT_HEADERS
-if "$(Configuration)" == "Debug" goto COPY_DEBUG_BUILT_HEADERS
-if "$(Configuration)" == "Debug_Broadway" goto COPY_DEBUG_BUILT_HEADERS
-
-:COPY_DEBUG_BUILT_HEADERS
-copy .\Debug\$(Platform)\obj\gdk-3\gdk\gdkconfig.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk
-copy .\Debug\$(Platform)\obj\gdk-3\gdk\gdkenumtypes.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk
-copy .\Debug\$(Platform)\obj\gdk-3\gdk\gdkversionmacros.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk
-copy .\Debug\$(Platform)\obj\gtk-3\gtk\gtktypebuiltins.h $(CopyDir)\include\gtk-$(ApiVersion)\gtk
-copy .\Debug\$(Platform)\obj\gtk-3\gtk\gtkversion.h $(CopyDir)\include\gtk-$(ApiVersion)\gtk
-goto CONTINUE_HEADERS
-
-:COPY_RELEASE_BUILT_HEADERS
-copy .\Release\$(Platform)\obj\gdk-3\gdk\gdkconfig.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk
-copy .\Release\$(Platform)\obj\gdk-3\gdk\gdkenumtypes.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk
-copy .\Release\$(Platform)\obj\gdk-3\gdk\gdkversionmacros.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk
-copy .\Release\$(Platform)\obj\gtk-3\gtk\gtktypebuiltins.h $(CopyDir)\include\gtk-$(ApiVersion)\gtk
-copy .\Release\$(Platform)\obj\gtk-3\gtk\gtkversion.h $(CopyDir)\include\gtk-$(ApiVersion)\gtk
-goto CONTINUE_HEADERS
-
-:CONTINUE_HEADERS
-#include "gdk-3.vs10.headers"
-#include "gdk3-win32.vs10.headers"
-
-#include "gtk-3.vs10.headers"
-
-mkdir $(CopyDir)\include\gail-$(ApiVersion)\libgail-util
-#include "gailutil-3.vs10.headers"
-
-
-for &#37;&#37;s in (16 22 24 32 48 256) do ((mkdir $(CopyDir)\share\icons\hicolor\&#37;&#37;sx&#37;&#37;s\apps) &amp; (copy /b ..\..\demos\gtk-demo\data\&#37;&#37;sx&#37;&#37;s\gtk3-demo.png $(CopyDir)\share\icons\hicolor\&#37;&#37;sx&#37;&#37;s\apps))
-
-mkdir $(CopyDir)\share\glib-2.0\schemas
-copy ..\..\gtk\org.gtk.Settings.ColorChooser.gschema.xml $(CopyDir)\share\glib-2.0\schemas
-copy ..\..\gtk\org.gtk.Settings.Debug.gschema.xml $(CopyDir)\share\glib-2.0\schemas
-copy ..\..\gtk\org.gtk.Settings.EmojiChooser.gschema.xml $(CopyDir)\share\glib-2.0\schemas
-copy ..\..\gtk\org.gtk.Settings.FileChooser.gschema.xml $(CopyDir)\share\glib-2.0\schemas
-copy ..\..\demos\gtk-demo\org.gtk.Demo.gschema.xml $(CopyDir)\share\glib-2.0\schemas
-    </GtkDoInstall>
-    <GtkDoInstallBroadwayHeaders>
-copy ..\..\gdk\broadway\gdkbroadway.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk
-mkdir $(CopyDir)\include\gtk-$(ApiVersion)\gdk\broadway
-#include "gdk3-broadway.vs10.headers"
-    </GtkDoInstallBroadwayHeaders>
-    <GtkPostInstall>
-echo "Compiling gsettings XML Files..."
-$(GlibEtcInstallRoot)\bin\glib-compile-schemas.exe $(CopyDir)\share\glib-2.0\schemas
-
-echo "Generating icon cache......"
-$(CopyDir)\bin\gtk-update-icon-cache.exe --ignore-theme-index --force "$(CopyDir)\share\icons\hicolor"
-    </GtkPostInstall>
-    <GenerateGtkPC>$(PythonDir)\python ..\gtkpc.py --prefix=$(CopyDir) --version=$(GtkVersion) --host=i686-pc-vs$(VSVer)</GenerateGtkPC>
-    <GenerateGtkPCX64>$(PythonDir)\python ..\gtkpc.py --prefix=$(CopyDir) --version=$(GtkVersion) --host=x86_64-pc-vs$(VSVer)</GenerateGtkPCX64>
-    <GtkPCFiles>..\gdk-3.0.pc;..\gtk+-3.0.pc;..\gail-3.0.pc</GtkPCFiles>
-  </PropertyGroup>
-  <PropertyGroup>
-    <_PropertySheetDisplayName>gtk3installsprops</_PropertySheetDisplayName>
-  </PropertyGroup>
-  <ItemGroup>
-    <BuildMacro Include="BinDir">
-      <Value>$(BinDir)</Value>
-    </BuildMacro>
-    <BuildMacro Include="InstalledDlls">
-      <Value>$(InstalledDlls)</Value>
-    </BuildMacro>
-    <BuildMacro Include="InstalledBins">
-      <Value>$(InstalledBins)</Value>
-    </BuildMacro>
-    <BuildMacro Include="InstalledBroadwayBins">
-      <Value>$(InstalledBroadwayBins)</Value>
-    </BuildMacro>
-    <BuildMacro Include="GtkDoInstallBin">
-      <Value>$(GtkDoInstallBin)</Value>
-    </BuildMacro>
-    <BuildMacro Include="GtkDoInstall">
-      <Value>$(GtkDoInstall)</Value>
-    </BuildMacro>
-    <BuildMacro Include="GtkDoInstallBroadwayHeaders">
-      <Value>$(GtkDoInstallBroadwayHeaders)</Value>
-    </BuildMacro>
-    <BuildMacro Include="GtkPostInstall">
-      <Value>$(GtkPostInstall)</Value>
-    </BuildMacro>
-    <BuildMacro Include="GenerateGtkPC">
-      <Value>$(GenerateGtkPC)</Value>
-    </BuildMacro>
-    <BuildMacro Include="GenerateGtkPCX64">
-      <Value>$(GenerateGtkPCX64)</Value>
-    </BuildMacro>
-    <BuildMacro Include="GtkPCFiles">
-      <Value>$(GtkPCFiles)</Value>
-    </BuildMacro>
-  </ItemGroup>
-</Project>
diff --git a/win32/vs9/Makefile.am b/win32/vs9/Makefile.am
index eded327..0619477 100644
--- a/win32/vs9/Makefile.am
+++ b/win32/vs9/Makefile.am
@@ -1,12 +1,5 @@
 include $(top_srcdir)/Makefile.decl
 
-MSVC_HEADERS_LISTS =	\
-	gdk3-win32.headers	\
-	gdk3-broadway.headers	\
-	gdk-3.headers		\
-	gtk-3.headers		\
-	gailutil-3.headers
-
 GENERATED_ITEMS = \
 	gdk3-win32.vcproj		\
 	gdk3-broadway.vcproj		\
@@ -46,10 +39,6 @@ EXTRA_DIST += \
 	gtk3-version-paths.vsprops.in	\
 	$(GENERATED_ITEMS)
 
-gtk3-install.vsprops: $(top_srcdir)/win32/vs9/gtk3-install.vspropsin $(MSVC_HEADERS_LISTS)
-	$(CPP) -P - <$(top_srcdir)/win32/vs9/gtk3-install.vspropsin >$@
-	rm $(MSVC_HEADERS_LISTS)
-
 DISTCLEANFILES = $(GENERATED_ITEMS)
 
 -include $(top_srcdir)/git.mk
diff --git a/win32/vs9/gtk3-install.vspropsin b/win32/vs9/gtk3-install.vspropsin
deleted file mode 100644
index 4fb97df..0000000
--- a/win32/vs9/gtk3-install.vspropsin
+++ /dev/null
@@ -1,208 +0,0 @@
-<?xml version="1.0" encoding="Windows-1252"?>
-<VisualStudioPropertySheet
-	ProjectType="Visual C++"
-	Version="8.00"
-	Name="gtkinstallprops"
-	InheritedPropertySheets=".\gtk3-build-defines.vsprops"
-	>
-	<UserMacro
-		Name="GtkDoInstallBin"
-		Value="
-mkdir $(CopyDir)\bin&#x0D;&#x0A;
-mkdir $(CopyDir)\share\gir-1.0&#x0D;&#x0A;
-mkdir $(CopyDir)\lib\pkgconfig&#x0D;&#x0A;
-mkdir $(CopyDir)\lib\girepository-1.0&#x0D;&#x0A;
-
-copy $(ConfigurationName)\$(PlatformName)\bin\$(GtkDllPrefix)gdk-3$(GtkDllSuffix).dll $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\$(GtkDllPrefix)gdk-3$(GtkDllSuffix).pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gdk-$(ApiVersion).lib $(CopyDir)\lib&#x0D;&#x0A;
-
-if &quot;$(ConfigurationName)&quot; == &quot;Release_Broadway&quot; goto DO_BROADWAY_BIN&#x0D;&#x0A;
-if &quot;$(ConfigurationName)&quot; == &quot;Debug_Broadway&quot; goto DO_BROADWAY_BIN&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\$(GtkDllPrefix)gtk-3$(GtkDllSuffix).dll $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\$(GtkDllPrefix)gtk-3$(GtkDllSuffix).pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\$(GtkDllPrefix)gailutil-3$(GtkDllSuffix).dll $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\$(GtkDllPrefix)gailutil-3$(GtkDllSuffix).pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk-$(ApiVersion).lib $(CopyDir)\lib&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gailutil-$(ApiVersion).lib $(CopyDir)\lib&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk3-demo.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk3-demo.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk3-demo-application.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk3-demo-application.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk3-icon-browser.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk3-icon-browser.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk-encode-symbolic-svg.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk-encode-symbolic-svg.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk-update-icon-cache.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk-update-icon-cache.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk-query-settings.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk-query-settings.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk-builder-tool.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\gtk-builder-tool.pdb $(CopyDir)\bin&#x0D;&#x0A;
-if exist $(ConfigurationName)\$(PlatformName)\bin\Gdk-3.0.gir copy $(ConfigurationName)\$(PlatformName)\bin\Gdk-3.0.gir $(CopyDir)\share\gir-1.0&#x0D;&#x0A;
-if exist $(ConfigurationName)\$(PlatformName)\bin\GdkWin32-3.0.gir copy $(ConfigurationName)\$(PlatformName)\bin\GdkWin32-3.0.gir $(CopyDir)\share\gir-1.0&#x0D;&#x0A;
-if exist $(ConfigurationName)\$(PlatformName)\bin\Gtk-3.0.gir copy $(ConfigurationName)\$(PlatformName)\bin\Gtk-3.0.gir $(CopyDir)\share\gir-1.0&#x0D;&#x0A;
-if exist $(ConfigurationName)\$(PlatformName)\bin\Gdk-3.0.typelib copy $(ConfigurationName)\$(PlatformName)\bin\Gdk-3.0.typelib $(CopyDir)\lib\girepository-1.0&#x0D;&#x0A;
-if exist $(ConfigurationName)\$(PlatformName)\bin\GdkWin32-3.0.typelib copy $(ConfigurationName)\$(PlatformName)\bin\GdkWin32-3.0.typelib $(CopyDir)\lib\girepository-1.0&#x0D;&#x0A;
-if exist $(ConfigurationName)\$(PlatformName)\bin\Gtk-3.0.typelib copy $(ConfigurationName)\$(PlatformName)\bin\Gtk-3.0.typelib $(CopyDir)\lib\girepository-1.0&#x0D;&#x0A;
-goto DONE_BIN&#x0D;&#x0A;
-
-:DO_BROADWAY_BIN&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\broadwayd.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy $(ConfigurationName)\$(PlatformName)\bin\broadwayd.pdb $(CopyDir)\bin&#x0D;&#x0A;
-
-if &quot;$(ConfigurationName)&quot; == &quot;Release_Broadway&quot; goto DO_BROADWAY_RELEASE&#x0D;&#x0A;
-if &quot;$(ConfigurationName)&quot; == &quot;Debug_Broadway&quot; goto DO_BROADWAY_DEBUG&#x0D;&#x0A;
-:DO_BROADWAY_RELEASE&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\$(GtkDllPrefix)gtk-3$(GtkDllSuffix).dll $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\$(GtkDllPrefix)gtk-3$(GtkDllSuffix).pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\$(GtkDllPrefix)gailutil-3$(GtkDllSuffix).dll $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\$(GtkDllPrefix)gailutil-3$(GtkDllSuffix).pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk-$(ApiVersion).lib $(CopyDir)\lib&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gailutil-$(ApiVersion).lib $(CopyDir)\lib&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk3-demo.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk3-demo.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk3-demo-application.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk3-demo-application.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk3-icon-browser.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk3-icon-browser.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk-encode-symbolic-svg.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk-encode-symbolic-svg.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk-update-icon-cache.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk-update-icon-cache.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk-query-settings.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk-query-settings.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk-builder-tool.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\bin\gtk-builder-tool.pdb $(CopyDir)\bin&#x0D;&#x0A;
-if exist .\Release\$(PlatformName)\bin\Gdk-3.0.gir copy .\Release\$(PlatformName)\bin\Gdk-3.0.gir $(CopyDir)\share\gir-1.0&#x0D;&#x0A;
-if exist .\Release\$(PlatformName)\bin\GdkWin32-3.0.gir copy .\Release\$(PlatformName)\bin\GdkWin32-3.0.gir $(CopyDir)\share\gir-1.0&#x0D;&#x0A;
-if exist .\Release\$(PlatformName)\bin\Gtk-3.0.gir copy .\Release\$(PlatformName)\bin\Gtk-3.0.gir $(CopyDir)\share\gir-1.0&#x0D;&#x0A;
-if exist .\Release\$(PlatformName)\bin\Gdk-3.0.typelib copy .\Release\$(PlatformName)\bin\Gdk-3.0.typelib $(CopyDir)\lib\girepository-1.0&#x0D;&#x0A;
-if exist .\Release\$(PlatformName)\bin\GdkWin32-3.0.typelib copy .\Release\$(PlatformName)\bin\GdkWin32-3.0.typelib $(CopyDir)\lib\girepository-1.0&#x0D;&#x0A;
-if exist .\Release\$(PlatformName)\bin\Gtk-3.0.typelib copy .\Release\$(PlatformName)\bin\Gtk-3.0.typelib $(CopyDir)\lib\girepository-1.0&#x0D;&#x0A;
-goto DONE_BIN&#x0D;&#x0A;
-
-:DO_BROADWAY_DEBUG&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\$(GtkDllPrefix)gtk-3$(GtkDllSuffix).dll $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\$(GtkDllPrefix)gtk-3$(GtkDllSuffix).pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\$(GtkDllPrefix)gailutil-3$(GtkDllSuffix).dll $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\$(GtkDllPrefix)gailutil-3$(GtkDllSuffix).pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk-$(ApiVersion).lib $(CopyDir)\lib&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gailutil-$(ApiVersion).lib $(CopyDir)\lib&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk3-demo.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk3-demo.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk3-demo-application.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk3-demo-application.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk3-icon-browser.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk3-icon-browser.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk-encode-symbolic-svg.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk-encode-symbolic-svg.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk-update-icon-cache.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk-update-icon-cache.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk-query-settings.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk-query-settings.pdb $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk-builder-tool.exe $(CopyDir)\bin&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\bin\gtk-builder-tool.pdb $(CopyDir)\bin&#x0D;&#x0A;
-if exist .\Debug\$(PlatformName)\bin\Gdk-3.0.gir copy .\Debug\$(PlatformName)\bin\Gdk-3.0.gir $(CopyDir)\share\gir-1.0&#x0D;&#x0A;
-if exist .\Debug\$(PlatformName)\bin\GdkWin32-3.0.gir copy .\Debug\$(PlatformName)\bin\GdkWin32-3.0.gir $(CopyDir)\share\gir-1.0&#x0D;&#x0A;
-if exist .\Debug\$(PlatformName)\bin\Gtk-3.0.gir copy .\Debug\$(PlatformName)\bin\Gtk-3.0.gir $(CopyDir)\share\gir-1.0&#x0D;&#x0A;
-if exist .\Debug\$(PlatformName)\bin\Gdk-3.0.typelib copy .\Debug\$(PlatformName)\bin\Gdk-3.0.typelib $(CopyDir)\lib\girepository-1.0&#x0D;&#x0A;
-if exist .\Debug\$(PlatformName)\bin\GdkWin32-3.0.typelib copy .\Debug\$(PlatformName)\bin\GdkWin32-3.0.typelib $(CopyDir)\lib\girepository-1.0&#x0D;&#x0A;
-if exist .\Debug\$(PlatformName)\bin\Gtk-3.0.typelib copy .\Debug\$(PlatformName)\bin\Gtk-3.0.typelib $(CopyDir)\lib\girepository-1.0&#x0D;&#x0A;
-:DONE_BIN&#x0D;&#x0A;
-
-copy ..\gdk-3.0.pc $(CopyDir)\lib\pkgconfig&#x0D;&#x0A;
-copy ..\gdk-3.0.pc $(CopyDir)\lib\pkgconfig\gdk-win32-3.0.pc&#x0D;&#x0A;
-copy &quot;..\gtk+-3.0.pc&quot; $(CopyDir)\lib\pkgconfig&#x0D;&#x0A;
-copy &quot;..\gtk+-3.0.pc&quot; &quot;$(CopyDir)\lib\pkgconfig\gtk+-win32-3.0.pc&quot;&#x0D;&#x0A;
-copy ..\gail-3.0.pc $(CopyDir)\lib\pkgconfig&#x0D;&#x0A;
-			  "
-	/>
-	<UserMacro
-		Name="GtkDoInstall"
-		Value="
-echo off&#x0D;&#x0A;
-mkdir $(CopyDir)\include\gtk-$(ApiVersion)\gdk\win32&#x0D;&#x0A;
-mkdir $(CopyDir)\include\gtk-$(ApiVersion)\gdk\deprecated&#x0D;&#x0A;
-mkdir $(CopyDir)\include\gtk-$(ApiVersion)\gtk\a11y&#x0D;&#x0A;
-mkdir $(CopyDir)\include\gtk-$(ApiVersion)\gtk\deprecated&#x0D;&#x0A;
-
-copy ..\..\gdk\win32\gdkwin32.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk&#x0D;&#x0A;
-
-if &quot;$(ConfigurationName)&quot; == &quot;Release&quot; goto COPY_RELEASE_BUILT_HEADERS&#x0D;&#x0A;
-if &quot;$(ConfigurationName)&quot; == &quot;Release_Broadway&quot; goto COPY_RELEASE_BUILT_HEADERS&#x0D;&#x0A;
-if &quot;$(ConfigurationName)&quot; == &quot;Debug&quot; goto COPY_DEBUG_BUILT_HEADERS&#x0D;&#x0A;
-if &quot;$(ConfigurationName)&quot; == &quot;Debug_Broadway&quot; goto COPY_DEBUG_BUILT_HEADERS&#x0D;&#x0A;
-
-:COPY_DEBUG_BUILT_HEADERS&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\obj\gdk-3\gdk\gdkconfig.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\obj\gdk-3\gdk\gdkenumtypes.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\obj\gdk-3\gdk\gdkversionmacros.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\obj\gtk-3\gtk\gtktypebuiltins.h $(CopyDir)\include\gtk-$(ApiVersion)\gtk&#x0D;&#x0A;
-copy .\Debug\$(PlatformName)\obj\gtk-3\gtk\gtkversion.h $(CopyDir)\include\gtk-$(ApiVersion)\gtk&#x0D;&#x0A;
-goto CONTINUE_HEADERS&#x0D;&#x0A;
-
-:COPY_RELEASE_BUILT_HEADERS&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\obj\gdk-3\gdk\gdkconfig.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\obj\gdk-3\gdk\gdkenumtypes.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\obj\gdk-3\gdk\gdkversionmacros.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\obj\gtk-3\gtk\gtktypebuiltins.h $(CopyDir)\include\gtk-$(ApiVersion)\gtk&#x0D;&#x0A;
-copy .\Release\$(PlatformName)\obj\gtk-3\gtk\gtkversion.h $(CopyDir)\include\gtk-$(ApiVersion)\gtk&#x0D;&#x0A;
-goto CONTINUE_HEADERS&#x0D;&#x0A;
-
-:CONTINUE_HEADERS&#x0D;&#x0A;
-#include "gdk-3.headers"
-#include "gdk3-win32.headers"
-#include "gtk-3.headers"
-
-mkdir $(CopyDir)\include\gail-$(ApiVersion)\libgail-util&#x0D;&#x0A;
-#include "gailutil-3.headers"
-
-mkdir $(CopyDir)\share\icons\hicolor\16x16\apps&#x0D;&#x0A;
-copy ..\..\demos\gtk-demo\data\16x16\gtk3-demo.png $(CopyDir)\share\icons\hicolor\16x16\apps\&#x0D;&#x0A;
-mkdir $(CopyDir)\share\icons\hicolor\22x22\apps&#x0D;&#x0A;
-copy ..\..\demos\gtk-demo\data\22x22\gtk3-demo.png $(CopyDir)\share\icons\hicolor\22x22\apps\&#x0D;&#x0A;
-mkdir $(CopyDir)\share\icons\hicolor\24x24\apps&#x0D;&#x0A;
-copy ..\..\demos\gtk-demo\data\24x24\gtk3-demo.png $(CopyDir)\share\icons\hicolor\24x24\apps\&#x0D;&#x0A;
-mkdir $(CopyDir)\share\icons\hicolor\32x32\apps&#x0D;&#x0A;
-copy ..\..\demos\gtk-demo\data\32x32\gtk3-demo.png $(CopyDir)\share\icons\hicolor\32x32\apps\&#x0D;&#x0A;
-mkdir $(CopyDir)\share\icons\hicolor\48x48\apps&#x0D;&#x0A;
-copy ..\..\demos\gtk-demo\data\48x48\gtk3-demo.png $(CopyDir)\share\icons\hicolor\48x48\apps\&#x0D;&#x0A;
-mkdir $(CopyDir)\share\icons\hicolor\256x256\apps&#x0D;&#x0A;
-copy ..\..\demos\gtk-demo\data\256x256\gtk3-demo.png $(CopyDir)\share\icons\hicolor\256x256\apps\&#x0D;&#x0A;
-
-mkdir $(CopyDir)\share\glib-2.0\schemas&#x0D;&#x0A;
-copy ..\..\gtk\org.gtk.Settings.ColorChooser.gschema.xml $(CopyDir)\share\glib-2.0\schemas&#x0D;&#x0A;
-copy ..\..\gtk\org.gtk.Settings.Debug.gschema.xml $(CopyDir)\share\glib-2.0\schemas&#x0D;&#x0A;
-copy ..\..\gtk\org.gtk.Settings.EmojiChooser.gschema.xml $(CopyDir)\share\glib-2.0\schemas&#x0D;&#x0A;
-copy ..\..\gtk\org.gtk.Settings.FileChooser.gschema.xml $(CopyDir)\share\glib-2.0\schemas&#x0D;&#x0A;
-copy ..\..\demos\gtk-demo\org.gtk.Demo.gschema.xml $(CopyDir)\share\glib-2.0\schemas&#x0D;&#x0A;
-		      "
-	/>
-	<UserMacro
-		Name="GtkPostInstall"
-		Value="
-echo &quot;Compiling gsettings XML Files...&quot;&#x0D;&#x0A;
-$(GlibEtcInstallRoot)\bin\glib-compile-schemas.exe $(CopyDir)\share\glib-2.0\schemas&#x0D;&#x0A;
-
-echo &quot;Generating icon cache...&quot;&#x0D;&#x0A;
-$(CopyDir)\bin\gtk-update-icon-cache.exe --ignore-theme-index --force &quot;$(CopyDir)\share\icons\hicolor&quot;
-		      "
-	/>
-	<UserMacro
-		Name="GtkDoInstallBroadwayHeaders"
-		Value="
-copy ..\..\gdk\broadway\gdkbroadway.h $(CopyDir)\include\gtk-$(ApiVersion)\gdk&#x0D;&#x0A;
-mkdir $(CopyDir)\include\gtk-$(ApiVersion)\gdk\broadway&#x0D;&#x0A;
-#include "gdk3-broadway.headers"
-        "
-	/>
-	<UserMacro
-		Name="GenerateGtkPC"
-		Value="$(PythonDir)\python ..\gtkpc.py --prefix=$(CopyDir) --version=$(GtkVersion) --host=i686-pc-vs$(VSVer)"
-	/>
-	<UserMacro
-		Name="GenerateGtkPCX64"
-		Value="$(PythonDirX64)\python ..\gtkpc.py --prefix=$(CopyDir) --version=$(GtkVersion) --host=x86_64-pc-vs$(VSVer)"
-	/>
-</VisualStudioPropertySheet>
